front-end/package.json

```diff
@@ -41,7 +41,7 @@
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.15.1",
     "jest-expo": "^35.0.0",
-    "react-test-renderer": "^16.12.0",
+    "react-test-renderer": "16.8",
```

Comment:
@JialinWang1 why is this change here?

Reply:
probably because of the merging conflicts, i had accepted the souce changes, i had now changed it to "react-test-renderer": "^16.12.0"